style.css

```diff
@@ -1,50 +1,84 @@
 :root {
-  --bg-color: #1f1d1d;
```

Comment:
nice CSS variables, you gave one by one for important ones yeah it really impressed me!